Vagrantfile

```
@@ -98,4 +98,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       end
   end
 
+  config.vm.define "focal", primary: true, autostart: true do |myhost|
+      myhost.vm.box = "ubuntu/focal64"
+      # myhost.vm.provision "shell", inline: "apt-get install -y python"
+      myhost.vm.provision "write_vbox_cfg", machine: "bionic"
+      myhost.vm.provision "ansible" do |ansible|
+        ansible.playbook = "playbook.yml"
+      end
```

Maybe we can dispense with write_vbox_cfg and just use this?

```suggestion
      myhost.vm.provision "ansible" do |ansible|
        ansible.playbook = "playbook.yml"
        ansible.groups = {
          "plone_servers" => ["focal"]
        }
      end
```